src/tailwind.css

```
@@ -51,6 +51,18 @@
   @apply tw:shrink-0 tw:rounded tw:p-1 tw:text-foreground tw:disabled:opacity-30 tw:hover:bg-muted/50;
 }
 
+@utility link-slot {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)